sdcm/fill_db_data.py

```diff
@@ -81,7 +87,7 @@ class FillDatabaseData(ClusterTester):
                                 firstname text,
                                 lastname text,
                                 age int
-                            ) WITH compaction = {'class': 'LeveledCompactionStrategy'};"""],
+                            ) WITH compaction = {'class': 'LeveledCompactionStrategy'}"""],
```

Comment:
just out of curiosity, why this compaction strategy was selected here?

Reply:
I think to cover same case with different CS.